NetworkingServiceKit/Reactive/NetworkManager+Reactive.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  NetworkManager+Reactive.swift
+//  Makespace Inc.
+//
+//  Created by Phillipe Casorla Sagot (@darkzlave) on 7/31/17.
+//
+//
+
+import Foundation
+import ReactiveSwift
+
+public protocol ReactiveNetworkManager : NetworkManager {
```

Comment:
Only comment I have is naming. I'd specifically go with a name like `ReactiveSwiftNetworkManager`, because I plan on adding another version of this for RXSwift, and reactive kind of just implies general FRP rather than the ReactiveSwift library specifically